ReadMe.md

```diff
@@ -15,7 +15,6 @@ More things to come:
 - Detect .fsx files without +x attrib.
 - Detect old versions of FSharpLint and fantomas/fantomless being used.
 - Detect unpinned versions, such as:
-    * Use of -latest in `runs-on:` GitHubCI tags.
```

Comment:
@realmarv you don't only need to remove this, you need to add a bullet point in other place of this file, like where it points to other .fsx scripts.

Reply:
Do I need to mention the type of unpinned version? (like "of -latest in `runs-on:` GitHubCI tags")
I assumed we are gonna have just a single script in the end to detect all types of unpinned versions

Reply:
> Do I need to mention the type of unpinned version?

If you don't mention it, then the reader of the ReadMe file might think that it's generic already, but this PR doesn't bring unpinned-versions generic detection, it only brings GitHubCI OS unpinned version. Do not change the ReadMe file to cover things that haven't been done yet.

Reply:
Done